README.md

```diff
@@ -23,7 +23,7 @@ The below is an excerpt from the types for the `datetime` module.
 ```
 MAXYEAR = ...  # type: int
 MINYEAR = ...  # type: int
-__doc__ = ...  # type: str
+__doc__ = ...  # type: Optional[str]
```

Comment:
Honestly I would remove all the dunder variables from the example, they don't add any clarity (and they're not in that file anyway).